src/components/pagination/pagination.stories.tsx

```diff
@@ -51,8 +51,47 @@ Basic.args = {
 export const Small = Template.bind( {} );
```

Comment:
Pagination `xs, lg` sizes are missing. Need to fix it from the component pop type.

Reply:
added